kaa/filetype/ini/__init__.py

```diff
@@ -0,0 +1,10 @@
+from kaa.filetype import filetypedef
+
+
+class FileTypeInfo(filetypedef.FileTypeInfo):
+    FILE_EXT = {'.ini', '.conf'}
```

Comment:
Can '.conf' files be able to considered ini files?  'http.conf' came into my mind at first.